src/Microsoft.OpenApi/Extensions/OpenApiTypeMapper.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using System;
+using System.Collections.Generic;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.OpenApi.Extensions
+{
+    /// <summary>
+    /// Extension methods for <see cref="Type"/>.
+    /// </summary>
+    public static class OpenApiTypeMapper
+    {
+        private static readonly Dictionary<Type, Func<OpenApiSchema>> _simpleTypeToOpenApiSchema = new()
```

Comment:
could we also add the opposite mapping? (from format/type to dotnet type) it'd make the life of generators much easier.
Here is a base for comparison.
https://github.com/microsoft/kiota/blob/8b211cd6a90d0211b5467728f7bb04a174bd9dee/tests/Kiota.Builder.Tests/KiotaBuilderTests.cs#L2714

Reply:
Yeah that makes sense.